task/buildah-oci-ta/0.1/buildah-oci-ta.yaml

```diff
@@ -274,29 +284,25 @@ spec:
       # Expose base image digests
       for image in $BASE_IMAGES; do
         if [ "${image}" != "scratch" ]; then
-          buildah images --format '{{ .Name }}:{{ .Tag }}@{{ .Digest }}' --filter reference="$image" >> $(results.BASE_IMAGES_DIGESTS.path)
+          buildah images --format '{{ .Name }}:{{ .Tag }}@{{ .Digest }}' --filter reference="$image" >>$( results.BASE_IMAGES_DIGESTS.path)
         fi
       done
 
       # Needed to generate base images SBOM
-      echo "$BASE_IMAGES" > /var/workdir/base_images_from_dockerfile
-
+      echo "$BASE_IMAGES" >/var/workdir/base_images_from_dockerfile
     securityContext:
       capabilities:
         add:
-          - SETFCAP
+        - SETFCAP
     volumeMounts:
     - mountPath: /var/lib/containers
       name: varlibcontainers
-    - mountPath: "/entitlement"
+    - mountPath: /entitlement
       name: etc-pki-entitlement
     workingDir: /var/workdir
-
-  - name: sbom-syft-generate
+  - computeResources: {}
     image: quay.io/redhat-appstudio/syft:v0.105.1@sha256:1910b829997650c696881e5fc2fc654ddf3184c27edb1b2024e9cb2ba51ac431
-    # Respect Syft configuration if the user has it in the root of their repository
```

Comment:
NIT: Any chance we can keep the comment?

Reply:
The issue here is that the YAML is unmarshalled to a golang struct and comments are lost, I missed this because the test code also does this, this might take some tinkering

Reply:
I had a try of restoring comments, the issue is that comments that are part of sequences loose their position as the tool adds or removes elements from the sequence, i.e. a comment on `spec.steps[n]` can't be guaranteed to be placed on the same place, and the option of tracking the comments by each item's identifier (i.e. step name) seemed like a lot of work. The attempt I had added comments to wrong places and for whatever reason (read: implementation bug on my side) did not restore this comment in particular. So the lazy option is to loose the comments, at least for now.